library/cwm/examples/example_helper.rb

```diff
@@ -0,0 +1,17 @@
+root_location = File.expand_path("../../../..", __FILE__)
```

Comment:
The helper should work if the binary is `ruby` instead of `y2base`, especially if we add
```rb
    if Yast.ui_component == ""
      Yast.ui_component = ARGV[0] || "ncurses"
    end
```
See also https://github.com/yast/yast-ruby-bindings/pull/142